### ISSUE TITLE
ArcFace Paddle Loss Nan

### ISSUE THREAD

**Issue by wario-wong** [deepinsight/insightface#1734]
Dear Sir,

When I trained the model with CASIA with batch size 256, the previous loss is around 5, the next loss becomes Nan.

**Comment by nttstar** [comment]
@littletomatodonkey

**Comment by wario-wong** [comment]
Thanks a lot Sir.
I seems solved this with a smaller learning rate where I set it to 0.01 instead of 0.1.

**Comment by littletomatodonkey** [comment]
Ok, the learning rate should be set relative smaller if your batch size is not so large.

**Comment by wario-wong** [comment]
Thanks for your reply Sir

**Comment by tessarwork** [comment]
hello how do you setting the learning rate?
